frontend/components/student/StudentView.tsx

```diff
@@ -215,7 +210,7 @@ const StudentView: React.FC<StudentViewProp> = ({
   useEffect(() => {
     setMotivation('');
     myStudent.statusSuggestions.forEach((suggestion) => {
-      if (suggestion.coachId == user.id) {
+      if (suggestion.suggester === user) {
```

Comment:
it is probably safer to compare suggestion.suggester.id and user.id since these objects won't necessarily have the same fields stored frontend.